src/packages/storage-provider/src/decorators/media-field.ts

```diff
@@ -0,0 +1,134 @@
+import {
+	BaseDataEntity,
+	Entity,
+	Field,
+	FieldOptions,
+	GraphQLEntity,
+	graphweaverMetadata,
+	FieldMetadata,
+} from '@exogee/graphweaver';
+import { S3StorageProvider } from '../storageProvider';
+import { Source } from 'graphql';
```

Comment:
We're exporting from `graphql/types` already from `@exogee/graphweaver`. Should we also export `Source` so people don't have to have their own `graphql` dependency to get in trouble?

Reply:
done 👍